README.md

```diff
@@ -177,8 +183,8 @@ less: {
     options: {
       paths: ["assets/css"],
       plugins: [
-        new (require('less-plugin-autoprefix'))({browsers: ["last 2 versions"]}),
-        new (require('less-plugin-clean-css'))(cleanCssOptions)
+        new require('less-plugin-autoprefix')({browsers: ["last 2 versions"]}),
```

Comment:
oops these changes should be in the build folder.. could you fix? else I can fix and you'll have to merge latest and rebuild README.